oemof/solph/blocks.py

```diff
@@ -245,62 +247,77 @@ class InvestmentFlow(SimpleBlock):
     **The following sets are created:** (-> see basic sets at
```

Comment:
Do we want to have all these Sets in the docstring? We don't have the sets in the docstring of the more "applied" components, which serve as docstring templates (like genericCHP, extractionTurbine,..), but I am not sure about the more fundamental blocks like flow etc.